datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -24,7 +24,7 @@ use std::{any::Any, sync::Arc};
 
 use arrow::array::*;
 use arrow::compute::kernels::arithmetic::{
-    add, add_scalar, divide, divide_scalar, modulus, modulus_scalar, multiply,
+    add, add_scalar, divide_opt, divide_scalar, modulus, modulus_scalar, multiply,
```

Comment:
default `divide` kernel changed behavior -- `divide_opt` now has the "NULL on divide by zero" behavior datafusion expects